packages/manager/apps/pci-kubernetes/src/components/region-selector/KubeDeploymentTile.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from 'clsx';
+import { ReactNode } from 'react';
+import RadioTile from '../radio-tile/RadioTile.component';
+
+export interface KubeDeploymentTileProps {
+  readonly title: string;
+  readonly pillLabel: ReactNode;
+  readonly description: string;
+  readonly isSelected?: boolean;
+  readonly onSelectedDeployment: () => void;
+}
+
+export function KubeDeploymentTile({
+  title,
+  pillLabel,
+  description,
+  onSelectedDeployment,
+  isSelected,
+}: KubeDeploymentTileProps) {
```

Comment:
Could be easier and prevent from forgetting marking each props as `readonly`: 

```suggestion
}: Readonly<KubeDeploymentTileProps>) {
```

Reply:
Why is it important to add Readonly ?

Reply:
It's to replace the `readonly` markers you add from each prop above ;)